addons/website_event_booth_sale/controllers/event_booth.py

```diff
@@ -9,12 +9,24 @@
 
 class WebsiteEventBoothController(WebsiteEventController):
 
+    def _prepare_booth_contact_form_values(self, event, booth_ids, booth_category_id):
```

Comment:
We usually put all routes together and then all utility methods, sorted alphabetically :socks: